src/tests/ftest/nvme/enospace.py

```diff
@@ -352,17 +350,15 @@ def test_performance_storage_full(self):
         # Write First
         self.start_ior_load(storage='SCM', operation="Auto_Write", percent=1)
         # Read the baseline data set
-        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1)
+        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1, create_cont=False)
```

Comment:
Due to `self.run_enospace_with_bg_job()` ending up calling `self.start_ior_load()` with the default `create_cont=True` we will need to save the container here and restore it before running the read again:
```suggestion
        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1, create_cont=False)
        baseline_container = self.nvme_local_cont
```